src/main/ruby/post-boot/post-boot.rb

```diff
@@ -10,75 +10,77 @@
   wd = Truffle::Boot.get_option('working_directory')
   Dir.chdir(wd) unless wd.empty?
 
-  # Always provided features: ruby --disable-gems -e 'puts $"'
-  begin
-    require 'enumerator'
-    require 'thread'
-    require 'rational'
-    require 'complex'
-    require 'unicode_normalize'
-    if Truffle::Boot.get_option 'patching'
-      Truffle::Boot.print_time_metric :'before-patching'
-      require 'truffle/patching'
-      Truffle::Patching.insert_patching_dir 'stdlib', "#{Truffle::Boot.ruby_home}/lib/mri"
-      Truffle::Boot.print_time_metric :'after-patching'
+  unless Truffle::Boot.ruby_home.nil?
```

Comment:
`if Truffle::Boot.ruby_home` is more direct and easier to read IMHO (`.nil?` is not idiomatic).